contracts-v0.2/deploy/Utils.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+import {Script, stdJson, VmSafe, console} from "forge-std/Script.sol";
+import {strings} from "solidity-stringutils/strings.sol";
+
+contract Utils is Script {
```

Comment:
I think we might have to reconsider the naming rules. In other contract scripts you added `S` suffix to specify that that contract is a script, but here you did not, maybe because it would like a bit weird (`UtilsS`)? Let's choose either `S` or `Script` suffix and use it everywhere.